commands/utils/params.go

```diff
@@ -85,9 +89,27 @@ func GetParamsAndClient() (configAggregator FrogbotConfigAggregator, server *cor
 		return nil, nil, nil, err
 	}
 
-	configData, err := ReadConfig(frogbotConfigPath)
+	var targetConfigContent []byte
+	var configData *FrogbotConfigAggregator
+	var missingConfigErr bool
+	if gitParams.GitProvider.String() == vcsutils.GitHub.String() || gitParams.GitProvider.String() == vcsutils.GitLab.String() {
```

Comment:
I think we can remove the "String()" 
```suggestion
	if gitParams.GitProvider == vcsutils.GitHub || gitParams.GitProvider == vcsutils.GitLab {
```

Reply:
done